packages/runtime-vapor/src/directives/vShow.ts

```diff
@@ -3,7 +3,7 @@ import type { ObjectDirective } from '../directive'
 const vShowMap = new WeakMap<HTMLElement, string>()
 
 export const vShow: ObjectDirective<HTMLElement> = {
-  beforeMount(node, { source: value }) {
+  beforeMount(node, { value }) {
```

Comment:
I think you should submit an issue to describe the problem first. This will help others locate the problem more accurately. What do you think?🧐
And when necessary, it is best to add unit tests